python/paddle/fluid/contrib/slim/quantization/quantize_program_pass.py

```diff
@@ -0,0 +1,175 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import logging
+import numpy as np
+from .... import core
+from ....framework import Program, Operator, Variable, program_guard
+from .... import unique_name
+from ....layer_helper import LayerHelper
+from ....param_attr import ParamAttr
+from ....initializer import Constant
+from ....log_helper import get_logger
+
+_logger = get_logger(
+    __name__, logging.INFO, fmt='%(asctime)s-%(levelname)s: %(message)s')
+
+
+class QuantizeProgramPass(object):
```

Comment:
`QuantizeProgramPass` --> `QuantizeTranspiler`.

Reply:
Add some notes for this class, such as 'it is just a patch for distributed quantization'.

Reply:
Done